src/main/scala/viper/silver/frontend/SilFrontEndConfig.scala

```diff
@@ -111,6 +111,13 @@ abstract class SilFrontendConfig(args: Seq[String], private var projectName: Str
     hidden = true
   )
 
+  val adtPlugin = opt[Boolean]("disableAdtPlugin",
```

Comment:
I know this is the same as the termination plugin above, but shouldn't this rather be `enableAdtPlugin`, `true` by default?

Reply:
@Aurel300 I don't have a strong opinion here. If you have, please bring it up in a Viper meeting. Or simply change the implementation when nobody is looking ;-)